test/xrp-client-integration-test.ts

```diff
@@ -0,0 +1,141 @@
+import { Wallet } from 'xpring-common-js'
+import { assert } from 'chai'
+import bigInt from 'big-integer'
+import XRPClient from '../src/xrp-client'
+import TransactionStatus from '../src/transaction-status'
+
+// A timeout for these tests.
+const timeoutMs = 60 * 1000 // 1 minute
+
+// An address on TestNet that has a balance.
+const recipientAddress = 'X7cBcY4bdTTzk3LHmrKAK6GyrirkXfLHGFxzke5zTmYMfw4'
```

Comment:
don't testnet addresses start with `T`?

Reply:
Yeah, technically. In reality, the prefix is just meta data that the client side SDKs attach. The actual rippled APIs only take in classic address / destination tag so everything is decoded before it's sent to the server anyway. 

We can change this to: T7X5peX2VRppBeaupFh5D6p1VHYZpgzKDzvThKxXh4rTBfq by using https://xrpaddress.info/ if we wanted.